tests/Integrations/CodeIgniter/V2_2/CommonScenariosTest.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+namespace DDTrace\Tests\Integrations\CodeIgniter\V2_2;
+
+use DDTrace\Tests\Common\SpanAssertion;
+use DDTrace\Tests\Common\WebFrameworkTestCase;
+use DDTrace\Tests\Frameworks\Util\Request\RequestSpec;
+use DDTrace\Type;
+
+final class CommonScenariosTest extends WebFrameworkTestCase
+{
+    const IS_SANDBOX = true;
+
+    protected static function getAppIndexScript()
+    {
+        return __DIR__ . '/../../../Frameworks/CodeIgniter/Version_2_2/ddshim.php';
+    }
+
+    protected static function getEnvs()
+    {
+        return array_merge(parent::getEnvs(), [
+            'DD_SERVICE_NAME' => 'codeigniter_test_app',
+        ]);
+    }
+
+    /**
+     * @dataProvider provideSpecs
+     * @param RequestSpec $spec
+     * @param array $spanExpectations
+     * @throws \Exception
+     */
+    public function testScenario(RequestSpec $spec, array $spanExpectations)
+    {
+        $traces = $this->tracesFromWebRequest(function () use ($spec) {
+            $this->call($spec);
+        });
+
+        $this->assertExpectedSpans($traces, $spanExpectations);
+    }
+
+    public function provideSpecs()
+    {
+        return $this->buildDataProvider(
+            [
+                'A simple GET request returning a string' => [
+                    SpanAssertion::build(
+                        'web.request',
+                        'codeigniter_test_app',
+                        'web',
+                        'GET /simple'
+                    )->withExactTags([
+                        'http.method' => 'GET',
+                        'http.url' => '/simple',
```

Comment:
`http.url` should be the full URL: `http://localhost:9999/simple`. I totally missed that in the last CR. Sorry about that.